items/JUNGLE_PICKAXE.json

```diff
@@ -1,17 +1,17 @@
 {
   "itemid": "minecraft:wooden_pickaxe",
-  "displayname": "§aJungle Pickaxe",
```

Comment:
While you are here could you add the hotm req to the jungle pickaxe (its hotm 3)

Reply:
Is is just the tooltip or someting else too?

Reply:
Just the tooltip